source/multigrid/mg_transfer_internal.cc

```diff
@@ -443,6 +443,24 @@ namespace internal
               fe.get_prolongation_matrix(c)(renumbering[j],renumbering[i]);
     }
 
+    namespace
+    {
+      /**
+       * helper function for setup_transfer
+       */
+      void replace_constrainted_dof_entries(const ConstraintMatrix &level_constraints,
+                                            std::vector<types::global_dof_index> &dof_indices)
+      {
+        for (unsigned int i=0; i<dof_indices.size(); ++i)
```

Comment:
not sure how is it harder to see given `auto &ind: dof_indices`, but YMMW. 

I think we can agree that it's (i) faster (ii) more concise.